src/app/cluster/list/external-cluster/template.html

```diff
@@ -69,47 +69,52 @@
             *matCellDef="let element"
             [attr.id]="'km-clusters-' + element.name">
           <div fxLayoutAlign=" center"
-               fxLayoutGap="8px">{{element.name}}
+               fxLayoutGap="8px">
+            <span>{{element.name}}</span>
+            <span *ngIf="(!element?.labels || element?.labels['is-imported'] !== 'true')"
```

Comment:
```suggestion
            <span *ngIf="(!element?.labels || element?.labels['is-imported'] === 'true')"
```

Reply:
It is fine! delete button will be shown only for **non-imported** clusters